docs/quickstart.md

```diff
@@ -2,198 +2,441 @@
 hide:
   - navigation
 ---
+## __Overview__
 
-This tutorial will introduce the main NeMoS functionalities. This is intended for users that are 
-already familiar with the GLM framework but want to learn how to interact with the NeMoS API. 
-If you have used [scikit-learn](https://scikit-learn.org/stable/) before, we are compatible with the [estimator API](https://scikit-learn.org/stable/modules/generated/sklearn.base.BaseEstimator.html), so the quickstart should look 
-familiar.
+NeMoS is a neural modeling software package designed to model neural spiking activity and other time-series data
+ powered by [JAX](https://jax.readthedocs.io/en/latest/). 
 
-In the following sessions, you will learn:
+At its core, NeMoS consists of two primary modules: the __`glm`__ and the __`basis`__ module.
 
-1. [How to define and fit a GLM model.](#basic-model-fitting)
-2. [What are the GLM input arguments.](#model-arguments)
-3. [How to use NeMoS with `pynapple` for pre-processing.](#pre-processing-with-pynapple)
-4. [How to use NeMoS with `scikit-learn` for pipelines and cross-validation.](#compatibility-with-scikit-learn)
+The **`glm`** module implements a Generalized Linear Model (GLM) to map features to neural activity, such as 
```

Comment:
it's annoying that the `glm` and `basis` here use `**` to make them bold, while above they use `__`. Minor point, but still makes reading the markdown a bit confusing.